views/js/qtiCreator/widgets/choices/helpers/formElement.js

```diff
@@ -8,9 +8,21 @@ define(['jquery'], function($) {
                 interaction = choice.getInteraction(),
                 $shuffleToggle = $container.find('[data-role="shuffle-pin"]');
 
+            var _toggleVisibility = function(show) {
+                if (show) {
+                    $shuffleToggle.show();
+                } else {
+                    $shuffleToggle.hide();
+                }
+                $('.qti-item').trigger('toolbarchange', {
+                    callee: 'formElementHelper'
+                });
+            };
+
             $shuffleToggle.off('mousedown').on('mousedown', function(e) {
-                e.stopPropagation();
                 var $icon = $(this).children();
+                e.stopPropagation();
```

Comment:
any specific reason to change the order? even related settings can be always on the very top

Reply:
Just to silence the ESLint error in this one file.